wlc/test_data/api/projects-hello-components--POST--739f9b01

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
The filename does not match what tests expects, either there is something wrong with handling file uploads in the tests and the hash is not stable, or you've changed the payload since introduction of the test.